test/support/pages/DiscoverPage.js

```diff
@@ -0,0 +1,199 @@
+// in test/support/pages/DiscoverPage.js
+define(function (require) {
+  // the page object is created as a constructor
+  // so we can provide the remote Command object
+  // at runtime
+  var Common = require('./Common');
+
+  var defaultTimeout = 20000;
+  var common;
+  var thisTime;
+
+  function DiscoverPage(remote) {
+    this.remote = remote;
+    common = new Common(this.remote);
+    thisTime = this.remote.setFindTimeout(defaultTimeout);
+  }
+
+  DiscoverPage.prototype = {
+    constructor: DiscoverPage,
+
+    clickTimepicker: function clickTimepicker() {
```

Comment:
Shouldn't this (and all the associated time picker stuff here) live in the header page helper?